homeassistant/components/plugwise/binary_sensor.py

```diff
@@ -96,3 +112,47 @@ def _async_process_data(self):
             self._icon = FLAME_ICON if self._is_on else IDLE_ICON
 
         self.async_write_ha_state()
+
+
+class PwNotifySensor(PwBinarySensor, BinarySensorEntity):
+    """Representation of a Plugwise Notification binary_sensor."""
+
+    def __init__(self, hass, api, coordinator, name, dev_id, binary_sensor, model):
+        """Set up the Plugwise API."""
+        super().__init__(api, coordinator, name, dev_id, binary_sensor, model)
+
+        self._binary_sensor = binary_sensor
+        self._hass = hass
+
+        self._is_on = False
+        self._icon = None
+        self._attributes = {}
+
+        self._unique_id = f"{dev_id}-{binary_sensor}"
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        return self._attributes
+
+    @callback
+    def _async_process_data(self):
+        """Update the entity."""
+        self._attributes = {}
+
+        notify = self._api.notifications
+
+        self._is_on = False
+        self._state = STATE_OFF
+        self._icon = NO_NOTIFICATION_ICON
+
+        if notify != {}:
+            self._is_on = True
+            self._state = STATE_ON
+            self._icon = NOTIFICATION_ICON
+
+            for dummy, details in notify.items():
+                for msg_type, msg in details.items():
+                    self._attributes[msg_type.upper()] = msg
```

Comment:
Please set the state attribute keys explicitly. We want to see what they are and the formatting.

Reply:
Added in e31f106